exporter/metric/cloudmonitoring.go

```diff
@@ -18,39 +18,17 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"time"
 
-	"go.opentelemetry.io/otel/sdk/metric/export"
-	"go.opentelemetry.io/otel/sdk/metric/export/aggregation"
-	"go.opentelemetry.io/otel/sdk/metric/sdkapi"
-	"go.opentelemetry.io/otel/sdk/resource"
+	metricapi "go.opentelemetry.io/otel/metric"
+	"go.opentelemetry.io/otel/metric/global"
+	"go.opentelemetry.io/otel/sdk/metric"
 
 	monitoring "cloud.google.com/go/monitoring/apiv3/v2"
 	"golang.org/x/oauth2/google"
 )
 
-const (
```

Comment:
For now, remove interval-related options, since exporters don't configure the interval anymore.